index.js

```diff
@@ -173,7 +183,7 @@ var getOmdb = function(id, type) {
     });
 };
 
-var getTmdb = function(id) {
+var getTmdb = function(id,type) {
```

Comment:
you should add somewhere 

`if (!type) type = 'movie';`

to allow backwards compatibility